src/main/kotlin/com/openlattice/chronicle/study/StudyApi.kt

```diff
@@ -44,4 +45,12 @@ interface StudyApi {
     fun getStudy(@Path(STUDY_ID) studyId: UUID): Study
 
 
+    /**
+     * Deletes an existing study based on id.
+     *
+     * @param studyId The id of the study to be destroyed.
+     * @return An integer corresponding to the number of destroyed studies
+     */
+    @DELETE(BASE + STUDY_ID_PATH)
+    fun destroyStudy(@Path(STUDY_ID) studyId: UUID): Int
```

Comment:
I don't think it's necessary to return an integer corresponding to the number of studies deleted. If the request returns with 200 OK, it's safe to assume the study was deleted.